RELEASES.md

```diff
@@ -1,3 +1,105 @@
+Version 1.23.0 (2018-01-04)
+==========================
+
+Language
+--------
+- [Fixed displaying duplicate errors on some trait bounds.][45772]
+
+Compiler
+--------
+- [Enabled `TrapUnreachable` in LLVM which should prevent some cases of
+  undefined behaviour.][45920]
+- [rustc now suggests renaming import if names clash.][45660]
+- [Display errors/warnings correctly when there are zero-width or
+  wide characters.][45711]
+- [rustc now uses subtyping on the left hand side of binary operations.][45435]
+  Which should fix some confusing errors in some operations.
+- [Bumped the minimum LLVM to 3.9][45326]
+- [rustc now avoids unnecessary copies of arguments that are
+  simple bindings][45380] This should improve memory usage on average by 5-10%.
+- [Updated musl used to build musl rustc to 1.1.17][45393]
+
+Libraries
+---------
+- [impl `From<T>` for `Mutex<T>` and `RwLock<T>`][46082]
+- [Allow a trailling comma in `assert_eq/ne` macro][45887]
+- [Implement Hash for raw pointers to unsized types][45483]
+- [impl `From<*mut T>` for `AtomicPtr<T>`][45610]
+- [impl `From<{number_size}>` for `Atomic{number_size}`][45610] eg. `From<u8>`
+  for `AtomicU8`.
+- [Removed the `T: Sync` requirement for `RwLock<T>: Send`][45267]
+- [Removed `T: Sized` requirement for `<*const T>::as_ref`
+  and `<*const T>::as_mut`][44932]
```

Comment:
This should be `<*const T>::as_ref`, `<*mut T>::as_ref`, and `<*mut T>::as_mut`.
(and not `<*const T>::as_mut`)